src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -161,16 +161,23 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
     }
 
     /**
-     * Opens the help window or focuses on it if it's already opened.
+     * Shows the list of possible commands on the right.
      */
     @FXML
     public void handleHelp() {
+        contactDisplay.showHelpDisplay();
+    }
```

Comment:
am not sure how to test for ui components

Reply:
not compulsory to test, so i set ignore the MainWindow file in codecov